src/git.js

```diff
@@ -99,6 +99,18 @@
       });
    };
 
+   /**
+    * Gets current working directory.
+    *
+    * @param {Function} [then]
+    * @returns {Git}
+    */
+   Git.prototype.getCwd = function (then) {
+      then(null, this._baseDir);
```

Comment:
Hi, in order to not break the `simple-git/promise` implementation, any extension to `simple-git` must be asynchronous - an example of which is in the current `.cwd` method above:

```
Git.prototype.getCwd = function (then) {
  var next = Git.trailingFunctionArgument(arguments);
  var workingDirectory = git._baseDir;

  return this.exec(function () {
    next && next(null, workingDirectory);
  });
};
```